examples/default.yaml

```diff
@@ -215,6 +215,18 @@ video:
   # on performance on macOS hosts: https://gitlab.com/qemu-project/qemu/-/issues/334
```

Comment:
Do we ever need VNC if we can just get this issue fixed?

Reply:
It is a separate feature, that bug seems specific to macOS ?